src/passes/functionPruner/functionRemover.ts

```diff
@@ -32,6 +40,13 @@ export class FunctionRemover extends ASTMapper {
     // Remove unreachable functions
     node.vFunctions
       .filter((func) => !this.reachableFunctions.has(func.id))
+      .filter(
+        (func) =>
+          !(
+            func instanceof CairoImportFunctionDefinition &&
+            func.functionStubKind === FunctionStubKind.TraitStructDefStub
+          ),
```

Comment:
Let's extract it to a named function

Reply:
I didn't get it, do you mean to split 46, 47 in two filter functions?

Reply:
done!